recipe/meta.yaml

```
@@ -50,8 +50,7 @@ requirements:
     - python
     - setuptools
     - fastrlock >=0.3
-    - numpy >=1.16
-    - six >=1.9.0
+    - numpy >=1.15
```

Looks like an older version of NumPy works here. Though maybe we had a reason for requiring a newer one 

https://github.com/cupy/cupy/blob/d49845a49f25f2d5675adf27b243aaf6bc2eb62f/setup.py#L44

---

I think that line is more likely an oversight (cc: @kmaehashi). CuPy v8 dropped the support for v1.15 (https://github.com/cupy/cupy/pull/3802), see also https://docs.cupy.dev/en/v8.5.0/install.html#python-dependencies.
```suggestion
    - numpy >=1.16
```